detectable/src/main/java/com/synopsys/integration/detectable/detectables/maven/cli/MavenCliExtractor.java

```diff
@@ -46,13 +49,27 @@ public Extraction extract(File directory, ExecutableTarget mavenExe, MavenCliExt
         List<String> includedScopes = mavenCliExtractorOptions.getMavenIncludedScopes();
         List<String> excludedModules = mavenCliExtractorOptions.getMavenExcludedModules();
         List<String> includedModules = mavenCliExtractorOptions.getMavenIncludedModules();
+        Boolean includeShadedDependencies = mavenCliExtractorOptions.getMavenIncludeShadedDependencies();
+
+        if(includeShadedDependencies) {
+            try {
+                mavenProjectInspectorDetectable.includeShadedDependencies = "include_shaded_dependencies";
+                mavenProjectInspectorDetectable.extractable();
+                Extraction extraction = mavenProjectInspectorDetectable.extract(extractionEnvironment);
+                shadedDependencies = mavenProjectInspectorDetectable.getShadedDependencies();
+            } catch (Exception e) {
+                throw new RuntimeException();
```

Comment:
It might be useful to either log some details on why the `Exception` happened, or include those details in the `RuntimeException`.